canvas_modules/common-canvas/src/common-properties/controls/checkboxset/checkboxset.scss

```diff
@@ -22,6 +22,18 @@
 		margin-bottom: $spacing-03;
 	}
 	.properties-checkboxset-container {
+		.checkbox-tooltip-container{
+			display: grid;
+			justify-content: center;
+			align-items: center;
+			grid-template-columns: 1fr 4fr;
+		}
+		.bx--checkbox-label-text{
+			margin-right: $spacing-04;
+		}
+		.tooltip-container{
```

Comment:
This really should be using
```
display: flex;
align-items: center;
```
If done on the correct div we don't need to manually adjust with margins.

Reply:
with height auto, tooltips got stacked side by side with their associated checkbox.